File: CompetitiveProgramming/DMOJ/Uncategorized/The Return of A Plus B/2019-04-28/TheReturnOfAPlusB.cpp
#include <iostream>
using namespace std;

int change(string s)
{
	if(s == "one" || s == "un" || s == "一")
		return 1;
	if(s == "two" || s == "deux" || s == "二")
		return 2;
	if(s == "three" || s == "trois" || s == "三")
		return 3;
	if(s == "four" || s == "quatre" || s == "四")
		return 4;
	if(s == "five" || s == "cinq" || s == "五")
		return 5;
	if(s == "six" || s == "六")
		return 6;
	if(s == "seven" || s == "sept" || s == "七")
		return 7;
	if(s == "eight" || s == "huit" || s == "八")
		return 8;
	if(s == "nine" || s == "neuf" || s == "九")
		return 9;
	if(s == "ten" || s == "dix" || s == "十")
		return 10;
	
	return stoi(s);
}

int main()
{
	int n;
	scanf("%d", &n);

	for(int i = 0; i < n; i++)
	{
		string a, b;
		cin >> a >> b;
		printf("%d\n", change(a) + change(b));
	}
	
	return 0;
}

File: CompetitiveProgramming/DMOJ/Uncategorized/The Return of A Plus B/2019-04-28/TheReturnOfAPlusB.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class TheReturnOfAPlusB
{
	static int convert(String c)
	{
		if(c.equals("zero") || c.equals("零") ) return 0;
		if(c.equals("one") || c.equals("un") || c.equals("一")) return 1;
		if(c.equals("two") || c.equals("deux") || c.equals("二")) return 2;
		if(c.equals("three") || c.equals("trois") || c.equals("三")) return 3;
		if(c.equals("four") || c.equals("quatre") || c.equals("四")) return 4;
		if(c.equals("five") || c.equals("cinq") || c.equals("五")) return 5;
		if(c.equals("six") || c.equals("六")) return 6;
		if(c.equals("seven") || c.equals("sept") || c.equals("七")) return 7;
		if(c.equals("eight") || c.equals("huit") || c.equals("八")) return 8;
		if(c.equals("nine") || c.equals("neuf") || c.equals("九")) return 9;
		if(c.equals("ten") || c.equals("dix") || c.equals("十")) return 10;
		return Integer.parseInt(String.valueOf(c));
	}
	
	public static void main (String [] args) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine());
		
		for(int i = 0; i < n; i++)
		{
			String [] data = br.readLine().split(" ");
			System.out.println(Integer.parseInt(data[0]) + Integer.parseInt(data[1]));
			//System.out.println(convert(data[0]) + convert(data[1]));
		}
	}
}